File: tests/fixtures/corpus/src/kotlin/Cases.kt
fun structured() {
    throw ApiException("KOTLIN_USER_MISSING", "User was not found", 404)
}

fun dynamic(dynamicCode: String) {
    throw ApiException(dynamicCode, "Dynamic code remains unresolved", 500)
}

fun generic() {
    throw IllegalStateException("Invalid state")
}

val ignored = IllegalStateException("KOTLIN_NOISE")

File: tests/fixtures/corpus/src/dart/cases.dart
void structured() {
  throw AppException('DART_USER_MISSING', 'User was not found');
}

void dynamic(String dynamicCode) {
  throw AppException(dynamicCode, 'Dynamic code remains unresolved');
}

void generic() {
  throw StateError('Invalid state');
}

final ignored = StateError('DART_NOISE');
